cdk/domino_cdk/domino_stack.py

```diff
@@ -31,7 +31,11 @@ def __init__(
             cdk.Tags.of(self).add(str(k), str(v))
 
         self.s3_stack = DominoS3Provisioner(self, "S3Stack", self.name, self.cfg.s3, nest)
-        self.vpc_stack = DominoVpcProvisioner(self, "VpcStack", self.name, self.cfg.vpc, nest)
+
+        self.vpc_stack = DominoVpcProvisioner(
+            self, "VpcStack", self.name, self.cfg.vpc, nest, flow_log_bucket=self.s3_stack.buckets.get("monitoring")
```

Comment:
Would it fail if monitoring_bucket is False in the config?

Reply:
At the moment, no it checks within DominoVpcProvisioner and raises. I'll add a test case.